src/java/htsjdk/samtools/fastq/FastqRecord.java

```diff
@@ -23,11 +23,13 @@
  */
 package htsjdk.samtools.fastq;
 
+import java.io.Serializable;
+
 /**
  * Represents a fastq record, fairly literally, i.e. without any conversion.
  */
-public class FastqRecord {
-
+public class FastqRecord implements Serializable,Cloneable {
```

Comment:
I think we should be wary about making things `Cloneable`.  `Clone` can be [problematic](http://www.javapractices.com/topic/TopicAction.do?Id=71).  

In general we prefer copy constructors (thanks for that!).